src/jobs/images.js

```diff
@@ -54,16 +54,32 @@ export default function (agenda) {
 			const refs = await getImageRefs();
 
 			logger.log('debug', `Checking updates for ${refs.length} images  in the registry`);
-
-			await Promise.all(refs.map(updateImage));
-
-			logger.log('debug', 'Done checking updates for images in the registry');
+			await checkImageRepoDigests(refs);
```

Comment:
It would be better to name it e.g. `updateImages` since it does more that check the digests.